### PR TITLE
new(ci): added manifests validation ci

### LINKED ISSUE #1035: Add a CI job to validate yaml configs
**Motivation**

It can happen that eg: a tab lingers in a commit, breaking an existing yaml, and then the master CI fails.

**Feature**

Add a CI job that uses eg: [`kubeval`](https://kubeval.instrumenta.dev/) to validate our config files.

### PR DESCRIPTION
**Pull request by cappellinsamuele**
Closes #1035

### CODE REVIEW

File: .github/workflows/validate_manifests.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    steps:
+      - name: Checkout test-infra ⤵️
+        uses: actions/checkout@v3
+
+      - name: Setup Kubeval ⛓️
+        uses: lra/setup-kubeval@v1.0.1
+        
+      - name: yaml config validation 🔍
+        run: kubeval --directories config/prow --ignore-missing-schemas
```
Comment (maxgio92) [inline .github/workflows/validate_manifests.yml:19]: Thanks @cappellinsamuele! Just missing a newline

**Review by maxgio92 — CHANGES_REQUESTED** [review]

File: .github/workflows/validate_manifests.yml (hunk 1 above)
Comment (cappellinsamuele) [reply .github/workflows/validate_manifests.yml]: Thank you!

**Review by FedeDP — APPROVED** [review]
/approve
Great job! Thank you!

**Comment by FedeDP** [comment]
Ops didn't see @maxgio92 requested changes. Sorry Max!

**Review by maxgio92 — APPROVED** [review]
/approve

No problem @FedeDP :-)

**Pull request merged**